shell/platform/darwin/ios/ios_gl_context.mm

```diff
@@ -22,6 +22,7 @@
     resource_context_.reset([[EAGLContext alloc] initWithAPI:kEAGLRenderingAPIOpenGLES2
                                                   sharegroup:context_.get().sharegroup]);
   }
+  [[NSNotificationCenter defaultCenter] postNotificationName:@"EAGLContextsharegroup" object:context_.get().sharegroup];
```

Comment:
What is this for?  It probably belongs in a separate PR.

Reply:
hi, this is belongs other PR, i'll remove it next commit

Reply:
done